src/components/panes/JoinSubmissionPane.jsx

```diff
@@ -12,19 +12,30 @@ import Button from '../misc/Button';
 import LoadingIndicator from '../misc/LoadingIndicator';
 import PaneBase from './PaneBase';
 import { getListItemById } from '../../utils/store';
+import { retrieveFieldTypesForOrganization } from '../../actions/personField';
 import {
     acceptJoinSubmission,
+    retrieveJoinForm,
     retrieveJoinSubmission,
 } from '../../actions/joinForm';
 import InfoList from '../misc/InfoList';
 
 
 const ADDR_FIELDS = [ 'co_address', 'street_address', 'zip_code', 'city', 'country' ];
 
-const mapStateToProps = (state, props) => ({
-    subItem: getListItemById(state.joinForms.submissionList,
-        props.paneData.params[0]),
-});
+const mapStateToProps = (state, props) => {
+    const subItem = getListItemById(state.joinForms.submissionList,
+        props.paneData.params[0]);
+
+    if (subItem) {
```

Comment:
If not subItem (i.e. when submissionList is not yet loaded), nothing is returned here and this is logged to the console:
`mapStateToProps() in Connect(InjectIntl(JoinSubmissionPane)) must return a plain object. Instead received undefined. warning.js:14:12`

Reply:
Will fix this when I get a chance.